<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="7,8">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>销售数据</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableWidget">
             <row>
              <property name="text">
               <string>A</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>B</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>C</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>D</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>E</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>F</string>
              </property>
             </row>
             <column>
              <property name="text">
               <string>1月</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>2月</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>3月</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>4月</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>5月</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>6月</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>统计数据</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableWidget_2">
             <row>
              <property name="text">
               <string>平均值</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>最大值</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>最小值</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>总和</string>
              </property>
             </row>
             <column>
              <property name="text">
               <string>A</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>B</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>C</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>D</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>E</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>F</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>刷新</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCustomPlot" name="widget" native="true"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>刷新</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <item>
                 <property name="text">
                  <string>一月</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>二月</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>三月</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>四月</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>五月</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>六月</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCustomPlot" name="widget_2" native="true"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="pushButton_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>刷新</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <item>
                 <property name="text">
                  <string>一月</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>二月</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>三月</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>四月</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>五月</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>六月</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCustomPlot" name="widget_3" native="true"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
